clang/lib/DPCT/APINamesWarp.inc

```diff
@@ -9,79 +9,94 @@
 // __all_sync
 SUBGROUPSIZE_FACTORY(
     UINT_MAX, MapNames::getClNamespace() + "all_of_group",
-    CALL_FACTORY_ENTRY(
-        "__all_sync",
-        CALL(
-            MapNames::getClNamespace() + "all_of_group", SUBGROUP,
-            makeCombinedArg(
-                ARG("(~"),
-                makeCombinedArg(
-                    makeCallArgCreatorWithCall(0),
-                    makeCombinedArg(
-                        makeCombinedArg(
-                            ARG(" & (0x1 << "),
-                            makeCombinedArg(
-                                SUBGROUP, ARG(".get_local_linear_id())) || "))),
-                        makeCallArgCreatorWithCall(1)))))))
+    ANALYZE_UNINIT_DEV_VAR_FACTORY(
+        CALL_FACTORY_ENTRY(
+            "__all_sync",
+            CALL(MapNames::getClNamespace() + "all_of_group", SUBGROUP,
+                 makeCombinedArg(
+                     ARG("(~"),
+                     makeCombinedArg(
+                         makeCallArgCreatorWithCall(0),
+                         makeCombinedArg(
+                             makeCombinedArg(
+                                 ARG(" & (0x1 << "),
+                                 makeCombinedArg(
+                                     SUBGROUP,
+                                     ARG(".get_local_linear_id())) || "))),
+                             makeCallArgCreatorWithCall(1)))))),
+        1))
```

Comment:
```suggestion
        1)) /*need add comment for the magic numbe 1*/
```

Reply:
I will move this argument before the inner factory.